packages/adapters/wagmi/src/client.ts

```diff
@@ -252,7 +252,17 @@ export class WagmiAdapter implements ChainAdapter {
   public construct(appKit: AppKit, options: AppKitOptionsWithCaipNetworks) {
     this.appKit = appKit
     this.options = options
-    this.defaultCaipNetwork = options.defaultNetwork || options.networks?.[0]
+    this.defaultCaipNetwork = CaipNetworksUtil.extendCaipNetwork(
```

Comment:
Even same for Wagmi's `construct` function, the `options.networks` are already extended